Vic2ToHoI4/Source/Hoi4Outputter/DifficultySettings.h

```diff
@@ -0,0 +1,13 @@
+#ifndef DIFFICULTY_SETTINGS_OUTPUTTER
```

Comment:
I've started emulating the naming scheme from EU4 to Vic2. Please rename this file to ```outDifficultySettings.h``` and have the .cpp file match.